test/extension.test.ts

```diff
@@ -5,8 +5,20 @@ import {ICoverageLines} from "../src/renderer";
 suite("Extension Tests", function() {
     this.timeout(25000);
 
+    test("Should start extension @integration", async () => {
```

Comment:
I actually removed this one as it was sometimes randomly failing due to startup times (annoying).